packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/en.ftl

```diff
@@ -1,6 +1,10 @@
 # Variables:
 #  $serviceName (String) - A service the user hasn't signed into before (e.g. Firefox)
-verifyLoginCode-subject-line = Sign-in code for { $serviceName }
-verifyLoginCode-title = Is this you signing in?
-verifyLoginCode-prompt-2 = If yes, here is the confirmation code:
+verifyLoginCode-subject-line-2 = Approve sign-in to { $serviceName }
+# Variables:
+#  $serviceName (String) - A service the user hasn't signed into before (e.g. Firefox)
+verifyLoginCode-title-2 = Did you sign in to { $serviceName }?
```

Comment:
nit: "sign-in" isn't hyphenated here, but it is above on L3 (which seems to be an existing usage).
Not sure which one matches our style guide, but hyphenated seems more common based on my quick 👀 through the diff.

Reply:
"sign in" = verb
"sign-in" = noun

This is correct.

Reply:
The usage in this file looks good to me. When used as a noun, sign in is typically hyphenated (approve the sign-in to your account) whereas when used as a verb (enter password to sign in to your account) it should not be hyphenated.